eventhubs/data-plane/pom.xml

```diff
@@ -1,79 +1,84 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-	<description>Java libraries for talking to Windows Azure Event Hubs</description>
-	
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+							 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	
 	<groupId>com.microsoft.azure</groupId>
 	<artifactId>azure-eventhubs-clients</artifactId>
-	<version>2.0.0</version>
 	<packaging>pom</packaging>
-	
+	<version>2.0.0</version>
+
+	<name>Microsoft Azure Event Hubs SDK Parent</name>
+	<description>Java libraries for talking to Windows Azure Event Hubs</description>
 	<url>https://github.com/Azure/azure-event-hubs</url>
-	
+
+	<parent>
+		<groupId>com.azure</groupId>
+		<artifactId>azure-client-sdk-parent</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+		<relativePath>../../pom.client.xml</relativePath>
+	</parent>
```

Comment:
Fix your editor - there are tab characters being added here - they should be converted to spaces.